backend/requirements.txt

```diff
@@ -14,6 +14,6 @@ platformdirs==2.5.2
 pymongo==4.3.2
 six==1.16.0
 tomli==2.0.1
-typing-extensions==4.3.0
+typing_extensions==4.3.0
```

Comment:
You should check which one of them is good - I hear about different malicious libs in pip with a really small change in name (as this above)